src/test/java/test/ignore/IgnoreTest.java

```diff
@@ -28,7 +28,7 @@ public void ignore_class_should_not_run_tests() {
 
     @Test
     public void ignore_class_with_inheritance_should_not_run_tests() {
-        TestNG tng = create(IgnoreClassParentSample.class);
+        TestNG tng = create(ChildSample.class);
```

Comment:
I wonder if the 2 classes are not 2 different use cases.

Reply:
Using `IgnoreClassParentSample` was just duplicating the scenario tested by `ignore_class_should_not_run_tests()`. That was why I altered the class.

Reply:
Ok 👍